Makefile

```diff
@@ -102,7 +102,8 @@ xgo:
 	go get github.com/karalabe/xgo
 
 generate: ##@other Regenerate assets and other auto-generated stuff
-	cp ./_assets/node_modules/web3/dist/web3.min.js ./static/scripts/web3.js
+	(cd _assets/static && npm install)
```

Comment:
Why is it wrapped in parentheses?

Reply:
you're right, that's not needed here. I'll remove it.

Reply:
Other than that, does it look OK?